src/tools/ScenarioMeasurement/Startup/Perfcollect.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.Diagnostics.Tracing.Parsers;
+using ScenarioMeasurement;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace ScenarioMeasurement
+{
+    public class Perfcollect : IDisposable
+    {
+        private readonly string filepath = Path.Combine(Directory.GetParent(Directory.GetCurrentDirectory()).FullName, @"Startup/perfcollect");
+        private ProcessHelper perfcollectProcess;
+        public string TraceName { get; private set; }
+        public string TraceDirectory { get; private set; }
+        public List<KernelKeyword> KernelEvents { get; set; }
+        public List<ClrKeyword> ClrEvents { get; set; }
+        public Perfcollect(string traceName, Logger logger) : this (traceName, Environment.CurrentDirectory, logger)
+        {
+        }
+
+        public Perfcollect(string traceName, string traceDirectory, Logger logger)
+        {
+            TraceName = traceName;
+/*            if (!File.Exists(filepath))
+            {
+                throw new FileNotFoundException($"Pefcollect not found at {filepath}. Please rebuild the project to download it.");
+            }*/
+
+            if (String.IsNullOrEmpty(traceName))
+            {
+                throw new ArgumentException("Trace file name cannot be empty.");
+            }
+
+
+            if (!Directory.Exists(traceDirectory))
+            {
+                Directory.CreateDirectory(traceDirectory);
+            }
+            TraceDirectory = traceDirectory;
+            
+
+            perfcollectProcess = new ProcessHelper(logger)
+            {
+                ProcessWillExit = true,
+                Executable = filepath,
+                Timeout = 300
+            };
+        }
+
+        public ProcessHelper.Result Start()
+        {
+            string arguments = $"start {TraceName} -events ";
+            
+            foreach (var keyword in KernelEvents)
+            {
+                arguments += Enum.GetName(typeof(KernelKeyword), keyword)+",";
+            }
```

Comment:
Why not just call keyword.ToString()?

Reply:
Didn't know that. Definitely better than GetName